products/firefox.md

```diff
@@ -9,6 +9,12 @@ changelogTemplate: https://www.mozilla.org/firefox/__LATEST__/releasenotes/
 LTSLabel: "<abbr title='Extended Support Release'>ESR</abbr>"
 releaseDateColumn: true
 
+identifiers:
+-   purl:  pkg:apk/alpine/firefox
+-   purl:  pkg:deb/debian/firefox
```

Comment:
Shouldn't this be firefox-esr ? There is nothing under https://packages.debian.org/bookworm/firefox.